docker/controller/Dockerfile

```diff
@@ -11,6 +11,8 @@ RUN apt-get update && apt-get -y install \
     software-properties-common \
     lsof \
     vim \
+    curl \
+    wget \
```

Comment:
small nit: Do we need both curl and wget here?

Reply:
no actually will remove `wget` as just curl is fine